main/src/ca/uwaterloo/flix/api/lsp/provider/completion/UseEnumCompleter.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022 Paul Butcher, Lukas Rønn
```

Comment:
???

Reply:
ah, I copied the template from my last file I believe.

Reply:
Roger. When you create the GitHub PR do take a look at the changes (on the files tab) then these things will be easy to spot 👍